frontend/js/mixins/inputFrame.js

```diff
@@ -37,7 +37,10 @@ export default {
 
       Object.keys(this.$store.state.form.errors).forEach((error) => {
         if (error.substr(0, error.indexOf('.')) === errorKeyWithoutLocale) {
-          locales.push(error.substr(error.indexOf('.') + 1, error.length))
+          const cleaned = error.substr(error.indexOf('.') + 1, error.length)
+          if (cleaned.length === 2) {
```

Comment:
Could this be a problem with regional locales like `fr_be`?

Reply:
True, Fixed this.